src/components/Gallery/ImageGallery.tsx

```diff
@@ -49,7 +49,7 @@ export class Gallery extends Component<GalleryProps, GalleryState> {
               <GridImage
                 key={asset.src}
                 selected={selectedAsset?.src === asset.src}
-                imageSrc={asset.src}
```

Comment:
It felt "cleaner" to pass the entire asset down instead of just having two props, `imageSrc` and `imageFilename`.

Reply:
I'm good with that 👍